java/arcs/sdk/android/storage/service/testutil/TestConnectionFactory.kt

```diff
@@ -45,9 +68,8 @@ fun TestConnectionFactorySingleService(ctx: Context) =
  * successfully bind with [StorageService] when using Robolectric.
  */
 class TestBindingDelegate(private val context: Context) : StorageServiceBindingDelegate {
-  private val serviceController by lazy {
+  private val serviceController =
     Robolectric.buildService(StorageService::class.java, null).create()
-  }
```

Comment:
To address the issues with calling create/destroy off of the main thread causing Robolectric to crash, I'm just creating the service immediately here (rather than doing it at the moment of the first onBind)